terragrunt/env/production/api/terragrunt.hcl

```diff
@@ -0,0 +1,11 @@
+terraform {
+  source = "../../../aws//api"
+}
+
+inputs = {
+  oidc_exists    = true
```

Comment:
This can be removed since we're getting rid of that variable.

Reply:
Removed the reference